src/libDirectoryService/Coinbase.cpp

```diff
@@ -397,19 +403,21 @@ void DirectoryService::InitCoinbase() {
   }
 
   uint128_t balance_left = total_reward - (suc_counter * reward_each) -
-                           (suc_lookup_counter * reward_each_lookup) +
-                           base_reward - (node_count * base_reward_each);
+                           (suc_lookup_counter * reward_each_lookup) -
+                           (node_count * base_reward_each);
```

Comment:
Should `base_reward_each` also rely on some success counter like `suc_base_counter` instead of `node_count`?